Cutelyst/Plugins/Utils/Validator/validator.cpp

```diff
@@ -57,36 +57,51 @@ void Validator::clear()
     }
 }
 
-bool Validator::validate(Context *c, ValidatorFlags flags) const
+Cutelyst::ValidatorResult Validator::validate(Context *c, ValidatorFlags flags) const
 {
+    ValidatorResult result;
+
     if (!c) {
```

Comment:
Hey, can we change this to:
    Q_ASSERT_X(!c, "validate", "No valid Context set, aborting validation.");
?